bin/acpp

```diff
@@ -2032,7 +2032,7 @@ if __name__ == '__main__':
       if not config.has_optimization_flag():
         print("acpp warning: No optimization flag was given, optimizations are "
               "disabled by default. Performance may be degraded. Compile with e.g. -O2/-O3 to "
-              "enable optimizations.")
+              "enable optimizations.", file=sys.stderr)
```

Comment:
I think we print warnings to stdout in more places than just here (you might also want to rebase on current develop). Probably those places should be changed too. Or even better, have shared functions to print warnings and errors.

Reply:
Done in #1340.